javascript/javascript1/week3/homework3.js

```diff
@@ -0,0 +1,153 @@
+////////////////////////////////////////////////////////////////////////////item array removal
+
+const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
+const nameToRemove = 'Ahmad';
+
+// Write some code here
+
+for (var i = 0; i < names.length; i++) {
+    if (names[i] === nameToRemove) {
+        names.splice(i, 1);
+    }
+}
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+/////////////////////////////////////////////////////////////////////////When we will be there
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function durationTime(infoArray) {
+
+    var hourse = Math.floor(infoArray.destinationDistance / infoArray.speed);
+    var minutes = Math.floor(((infoArray.destinationDistance / infoArray.speed) - hourse) * 60);
+    return hourse + ' hourse and ' + minutes + ' minutes';
+
+}
+const travelTime = durationTime(travelInformation);
+console.log(travelTime);
+
+
+////////////////////////////////////////////////////////////////////Series duration of my life
+
+const seriesDurations = [{
+        title: 'Friends',
+        days: 4,
+        hours: 2,
+        minutes: 20,
+    },
+    {
+        title: 'How I met your mother',
+        days: 7,
+        hours: 6,
+        minutes: 36,
+    },
+    {
+        title: 'Lost',
+        days: 11,
+        hours: 0,
+        minutes: 36,
+    }
+]
+
+let total = 0;
+
+function logOutSeriesText() {
+    for (let i = 0; i < seriesDurations.length; i++) {
+        var timeInMinutes = ((seriesDurations[i].days) * 24 * 60) + ((seriesDurations[i].hours) * 60) + seriesDurations[i].minutes;
+        console.log(seriesDurations[i].title + ' took ' + ((timeInMinutes / (80 * 356 * 24 * 60)) * 100).toFixed(2) + '% of my life.');
+        total += Number(((timeInMinutes / (80 * 356 * 24 * 60)) * 100).toFixed(2));
+
+    }
+    console.log(`In total that is ${total}% of my life.`)
+}
+
+logOutSeriesText(); // logs out the text found above
+
+
+//////////////////////////////////////////////////////////////////////////////////save a note
+
+const notes = [];
+
+function saveNote(content, id) {
+    notes.push({
+        content,
+        id
+    });
+}
+
+saveNote('Pick up groceries', 1);
+saveNote('Do laundry', 2);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+function getNote(id) {
+    for (let i = 0; i < notes.length; i++) {
+        if (id === '' || id === isNaN() || isNaN(id)) {
+            return 'Error: Please write a number';
+        } else if (notes[i].id === id) {
+            return 'content: ' + notes[i].content + ', id: ' + id;
+        }
+    }
+};
+
+const firstNote = getNote(2);
+console.log(firstNote); // {content: 'Pick up groceries', id: 1}
```

Comment:
This isn't the first note, remember? One [LC84-85](https://github.com/hamedveloper/hyf-homework/pull/8/files#diff-1381b4fe1e525f3ab6dda814cef242cdR84-R85), you added two notes. Calling `getNote(2)` will return the item with id matching `2`